src/backend/aspen/workflows/import_locations/save.py

```diff
@@ -18,12 +18,16 @@ def save():
 
     with session_scope(interface) as session:
         # Insert all locations from gisaid_metadata
-        gisaid_locations_select = sa.select(
-            GisaidMetadata.region,
-            GisaidMetadata.country,
-            GisaidMetadata.division,
-            GisaidMetadata.location,
-        ).distinct()
+        gisaid_locations_select = (
+            sa.select(
+                GisaidMetadata.region,
+                GisaidMetadata.country,
+                GisaidMetadata.division,
+                GisaidMetadata.location,
+            )
+            .where(GisaidMetadata.location != "")
```

Comment:
My SQL is weak, so I really don't know -- do we have to worry about this still selecting on rows where location is NULL? Even if there are no NULLs in location right now, we might want to add an additional clause to protect against it just in case Gisaid changes how they represent nothingness later

Reply:
Added!